compiler/rustc_mir_transform/src/lib.rs

```diff
@@ -509,7 +511,6 @@ fn run_runtime_lowering_passes<'tcx>(tcx: TyCtxt<'tcx>, body: &mut Body<'tcx>) {
     let passes: &[&dyn MirPass<'tcx>] = &[
         // These next passes must be executed together
         &add_call_guards::CriticalCallEdges,
-        &elaborate_drops::ElaborateDrops,
```

Comment:
The other pass should run just before the drop elaboration still. (It breaks critical edges and makes it possible to execute extra code when a function call returns by inserting code to a destination block.)